### PR TITLE
Change name to Open Liberty Tools

### PR DESCRIPTION
**Pull request by kathrynkodama**
Signed-off-by: Kathryn Kodama <kathryn.s.kodama@gmail.com>

### CODE REVIEW

**Comment by yeekangc** [comment]
May be a better description for "Liberty Dev Dashboard explorer on the side bar" if possible.

**Comment by kathrynkodama** [comment]
> May be a better description for "Liberty Dev Dashboard explorer on the side bar" if possible.

How about : `Through the Liberty Dev Dashboard explorer view, you can start, stop, or interact with dev mode on all available Liberty Maven projects in your workspace.`

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Liberty Dev Mode for VS Code
-A VS Code extension for Liberty dev mode. The extension will detect your Liberty Maven project if it detects the `io.openliberty.tools:liberty-maven-plugin` in the `pom.xml`. Through the Liberty Dev Dashboard explorer on the side bar, you can start, stop, or interact with dev mode on all available Liberty dev projects in your workspace.
+# Open Liberty Tools for VS Code
+A VS Code extension for Open Liberty. The extension will detect your Liberty Maven project if it detects the `io.openliberty.tools:liberty-maven-plugin` in the `pom.xml`. Through the Liberty Dev Dashboard explorer on the side bar, you can start, stop, or interact with Liberty dev mode on all available Liberty Maven projects in your workspace.
```
Comment (ericglau) [inline README.md]: `Liberty dev mode` could link to the dev mode doc at https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev

Comment (kathrynkodama) [reply README.md]: Created a new pull request to incorporate this change: https://github.com/OpenLiberty/liberty-dev-vscode-ext/pull/16

**Pull request merged**